felix/cachingmap/caching_map.go

```diff
@@ -137,22 +86,7 @@ func (c *CachingMap[K, V]) SetDesired(k K, v V) {
 	if logrus.GetLevel() >= logrus.DebugLevel {
 		logrus.WithFields(logrus.Fields{"name": c.name, "k": k, "v": v}).Debug("SetDesired")
```

Comment:
this is going to print the log 2x, once with the cachimg map name and then without. I wonder if it would be nicer to pass logging context to the delta tracker and embed it. Some of these functions could then go.

Reply:
I decided not to embed it; just because I don't want to expose the ReplaceXXX method, which is used by the caching map. I suppose I could try to be clever and have an embedded interface with the methods I want to expose, that points to the delta tracker but... I think that'd hurt readability.